force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -29,7 +29,10 @@ public with sharing class FlowControlLoopsCollections {
      * @return "Hello World!" if x greater than y, otherwise return null.
      */
     public static String helloWorld(Integer x, Integer y) {
-        return null; // Replace null with the variable you used to store the result
+        if (x > y) {
+            return 'Hello World!';
+        }
+        return null;
```

Comment:
I would probably prefer to do this so that I can return "invalid" input earlier, but this is mostly preference/convention.
```suggestion
        if (x <= y) {
            return null;
        }
        return 'Hello World!';
```